.github/workflows/flake8.yml

```diff
@@ -0,0 +1,21 @@
+
+name: flake8
+
+on: push
+
+jobs:
+  flake8:
+    runs-on: ubuntu-latest
+    name: flake8
+    steps:
+      - name: Check out source
+        uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
+      - name: flake8
+        uses: py-actions/flake8@v2
+        with:
+          max-line-length: 120
+          exclude: "nwbinspector/__init__.py"
```

Comment:
I think it would be better to set this up in `setup.cfg` and have this Action use `setup.cfg` if it does not by default. Then, devs can run `flake8` locally using the same config.